src/Movie-details/MovieDetails.js

```diff
@@ -43,20 +43,32 @@ class MovieDetails extends Component {
       releaseDate: this.formatDate(movie.release_date),
       overview: movie.overview,
       averageRating: movie.average_rating,
-      genres: movie.genres,
+      genres: this.formatGenres(movie.genres),
       budget: movie.budget,
       revenue: movie.revenue,
-      runtime: movie.runtime,
+      runtime: this.formatRuntime(movie.runtime),
       tagline: movie.tagline,
     });
   }
 
+  formatGenres(genres) {
+    return genres.map(
+      (genre, i) => !i ? ` ${genre} ` : `/ ${genre} `
```

Comment:
THIS IS RAD! I've never used the second argument for map. THIS IS HANDY!!! Love it.